build.gradle

```diff
@@ -107,8 +107,8 @@ task bundledJar(type: Jar) {
 
 jacocoTestReport {
 	reports {
-		xml.enabled = true
-		html.enabled = true
+		xml.required = true
```

Comment:
To be compatible with future releases, see [the docs](https://docs.gradle.org/7.3/dsl/org.gradle.api.reporting.Report.html#org.gradle.api.reporting.Report:enabled).